CodenameOne/src/com/codename1/ui/SwipeableContainer.java

```diff
@@ -383,6 +383,8 @@ public void actionPerformed(ActionEvent evt) {
                     }
 
                     if (initialX != -1) {
+                        if (getPreviouslyOpened() != null && getPreviouslyOpened() != SwipeableContainer.this && getPreviouslyOpened().isOpen())
+                            getPreviouslyOpened().close();

```

Comment:
Can you add curly brackets around the statement?

Reply:
Thanks, yes, sure